### DISCUSSION TITLE
Certificate renewal operations for certificate running outside the cluster

### DISCUSSION THREAD

**Discussion by poojagupta1418** [Keyfactor/ejbca-ce#156] [Q&A]
Hi, 

I am running openshift cluster and implement EJBCA as a container(bitnami helm chart) , now i would like to handle certificate autorenewal operations for RHEL nodes running outside the cluster .
Is there any way to renew certificate generated outside the cluster,

Kindly share the inputs and if it possible please guide and share the reference for the same

**Comment by primetomas** [comment]
You can find lots of documentation how to use different standard protocols and APIs to enroll and renew certificates (as well as manual methods of course) in the documentation.
https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/protocols
